programs/sbf/Cargo.toml

```diff
@@ -64,6 +64,10 @@ license = { workspace = true }
 edition = { workspace = true }
 
 [features]
+default = [
+    "sbf_c",
+    "sbf_rust",
+]
```

Comment:
why not? :) Otherwise, we have to leak these impl-detail features to `./ci/test-checks.sh` by starting manually unrolling `cargo-for-all-lock-files.sh`.